edx_name_affirmation/signals.py

```diff
@@ -0,0 +1,179 @@
+# pylint: disable=logging-format-interpolation
+
+""""
+Signal handlers for IDV and proctoring service signals
+"""
+
+import logging
+
+from django.contrib.auth import get_user_model
+
+from edx_name_affirmation.models import VerifiedName
+from edx_name_affirmation.statuses import VerifiedNameStatus
+from edx_name_affirmation.toggles import is_verified_name_enabled
+
+try:
+    from lms.djangoapps.verify_student.signals import idv_update_signal
```

Comment:
Could you add comments here to explain why the import might not be successful?

Reply:
I know this is an import but it's a weird one that exists only to enable the connect that is waaaay at the other end of the file. Would it make sense to combine them at the end of the file and do

try:
  import
  connect
except:
  pass (or log?)

I'm guessing this is organized this way to avoid direct import of proctoring etc into name affirmation but connection when they are both imported into the platform